From 920014ff93fc05f9d1dd2c99fadb48e8785e5233 Mon Sep 17 00:00:00 2001
From: richfifeg <123514542+richfifeg@users.noreply.github.com>
Date: Mon, 30 Dec 2024 10:07:02 -0800
Subject: [PATCH] 60-sensor.hwdb: Loosen model mask to include entire series

This sensor is the same for Asus's entire Q5##L series, so match accordingly.

Tested with my Q502LAB.

(cherry picked from commit 89c4fe6c211d5e9cb380329f638609dec26bed0a)

Resolves: RHEL-49810
---
 hwdb.d/60-sensor.hwdb | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 02f4041475..3379294a08 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -164,8 +164,7 @@ sensor:modalias:acpi:INVN6500*:dmi:*svn*ASUSTeK*:*pn*TP300LA:*
 sensor:modalias:acpi:INVN6500*:dmi:*svn*ASUSTeK*:*pn*TP300LD:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
 
-sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pn*Q551LB:*
-sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pn*Q551LN:*
+sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pn*Q5??L*:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; -1, 0, 0; 0, 0, 1
 
 sensor:modalias:acpi:KXJ2109*:dmi:*:svnASUSTeK*:pnME176C:*
